### PR TITLE
fix(sonar bugs): fixed sonar bugs errors

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**
## Description

Sonar cloud  one keyboard listener fixes

## Why

Code quality improvements

## Issue

n/a

## Checklist

Please delete options that are not relevant.

- [X] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [X] I have performed [IP checks](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool) for added or updated 3rd party libraries
- [ ] I have created and linked IP issues or requested their creation by a committer
- [X] I have performed a self-review of my own code
- [X] I have successfully tested my changes locally
- [ ] I have added tests that prove my changes work
- [ ] I have checked that new and existing tests pass locally with my changes
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/overlays/IDPDelete/index.tsx [hunk 1]
```diff
@@ -70,7 +70,13 @@ const getIntro = (status: Status, retry: () => void) => {
     return (
       <Trans i18nKey="[span]overlays.idp_delete_error_intro">
         Please{' '}
-        <span className="idp-retry-link" onClick={retry}>
+        <span
+          className="idp-retry-link"
+          onClick={retry}
+          onKeyUp={() => {
+            //doChange
```
Comment (oyo) [inline src/components/overlays/IDPDelete/index.tsx:77]: this function doesn't to anything - please change this (and all the other ones in this PR) to the comment we have used in those cases before

`// do nothing`

**Review by oyo — CHANGES_REQUESTED** [review]
just some re-wording to be consistent across the application

File: src/components/overlays/IDPDelete/index.tsx (hunk 1 above)
Comment (nidhigarg-bmw) [reply src/components/overlays/IDPDelete/index.tsx:77]: @oyo This is solution wrt this ticket https://jira.catena-x.net/browse/CPLP-3488
I found this as a solution. What you think ?

Comment (oyo) [reply src/components/overlays/IDPDelete/index.tsx:77]: @nidhigarg-bmw I think it doesn't really matter what comment we write but it should be understandable to the reviewer. `doChange` is somehow irritating when actually nothing happens. The eslint example for this rule was `// do nothing` which seems more appropriate to me and also we used this for empty functions in the past. That's why I'd like to stick with that comment.

**Review by oyo — APPROVED** [review]

**Pull request merged**